src/components/Header/index.tsx

```diff
@@ -291,17 +291,21 @@ function Header() {
             {networkWithoutSWPR && <HeaderLinkBadge label="NOT&nbsp;AVAILABLE" />}
           </HeaderLink>
           <HeaderLink data-testid="bridge-nav-link" id="bridge-nav-link" to="/bridge">
-            {t('bridge')}
-            <HeaderLinkBadge label="BETA" />
+            <>
+              {t('bridge')}
+              <HeaderLinkBadge label="BETA" />
+            </>
```

Comment:
Was wondering... Why doesn't `@types/react` allow multiple child in this new version? And when declaring children prop should we allow it?

Reply:
https://stackoverflow.com/questions/62382324/react-typescript-this-jsx-tags-children-prop-expects-a-single-child-of-type

Reply:
I will see if we can put a global type to fix this. 👍🏼